packages/plugin/src/Bundles/Backup/Export/FormieV3/Fields/Processors/AbstractFieldProcessor.php

```diff
@@ -0,0 +1,99 @@
+<?php
+
+namespace Solspace\Freeform\Bundles\Backup\Export\FormieV3\Fields\Processors;
+
+use Solspace\Freeform\Bundles\Backup\DTO\Field;
+use Solspace\Freeform\Bundles\Backup\Export\FormieV3\Fields\Interfaces\FieldProcessorInterface;
+use Solspace\Freeform\Library\Helpers\HashHelper;
+
+abstract class AbstractFieldProcessor implements FieldProcessorInterface
+{
+    public function process($formField, string $formUid, int $index): ?Field
+    {
+        $field = new Field();
+        $field->uid = $formField->uid ?? HashHelper::sha1($formUid.'field'.$index, 32);
+        $field->name = $formField->label ?? 'Field '.($index + 1);
+        $field->handle = $this->getFieldHandle($formField->handle ?? 'field'.$index);
+        $field->type = $this->getFreeformFieldClass();
+        $field->required = $formField->required ?? false;
+        $field->metadata = $this->getFieldMetadata($formField);
+
+        return $field;
+    }
+
+    protected function getFieldHandle($currentHandle): string
```

Comment:
we're missing some type safety here

```suggestion
    protected function getFieldHandle(string $currentHandle): string
```